### DISCUSSION TITLE
Close to system tray

### DISCUSSION THREAD

**Discussion by CalfMoon** [i3/i3#5942] [Q&A]
This might be a stupid question but, i need to hide some programs in my system tray when closing them.
The main program i want right now is obs. I can open it in system tray but when i open the gui and close it, it closes even in the system tray, is there a way to send it back to only system tray?
Currently i use $mod+q which is bound to kill is there another such command for hiding in system tray?

Edit: So i just noticed steam does what i want obs to do, stay in system tray when i close it and obs has the same option but for minimizing, but there is no minimizing in i3, is there still a way to accomplish what i want?

**Comment by orestisfl** [comment]
This is usually up to the application, is there any option?

**Reply by CalfMoon** [reply]
Yes there is an option in OBS and this is what happens when I turn on 'minimize to system tray instead of taskbar' and minimize it in another DE(cinnamon in this case)
